CONTRIBUTING.rst

```diff
@@ -107,15 +107,115 @@ category. You can quickly list these by searching the Red Hat bugzilla for bugs
 
 Patches for bugs without keywords are welcome, too!
 
+WebUI development
+-----------------
+
+For good starting points how to develop and test Web UI interface of Anaconda see ``ui/webui/README.rst`` and ``ui/webui/test/README.rst``.
+
+PatternFly
+^^^^^^^^^^
+
+Provides all the Web UI Widgets we use.
+
+https://www.patternfly.org/v4/
+
+NOTE: Right now we are using PatternFly 4 & version 5 is about to be released. Please take this into account when referencing the documentation.
+
+Pixel tests
```

Comment:
There is already a section about pixel tests here https://github.com/rhinstaller/anaconda/tree/master/ui/webui/test#pixel-tests

Webui  testing should be documented in https://github.com/rhinstaller/anaconda/tree/master/ui/webui/test/README.rst in general. I dont see why you want to introduce a duplicated documentation here.

Reply:
how-to-parts should be  definitely in deeper level READMEs instead of the top level CONTROBUTING.rst file. 

Generally I think that top level file should act as  a a pointer  and never contain duplicate information (see pixel tests explanation)